THIRD-PARTY

```diff
@@ -902,27 +900,33 @@ POM License: Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-20. Group: com.google.errorprone  Name: error_prone_annotations  Version: 2.11.0
+20. Group: com.google.errorprone  Name: error_prone_annotations  Version: 2.10.0
 
 POM License: Apache 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
-21. Group: com.google.errorprone  Name: error_prone_annotations  Version: 2.7.1
+21. Group: com.google.errorprone  Name: error_prone_annotations  Version: 2.11.0
```

Comment:
Is this duplicate of line 903 ? How does `./gradlew generateThirdPartyReport` resolve if there are duplicate in this case ?